UI/GardenDeskForm.Designer.cs

```diff
@@ -1455,12 +1473,12 @@ private void InitializeComponent()
             // 
             // btnCancelChangesEmployee
             // 
-            btnCancelChangesEmployee.BackColor = Color.DarkGray;
+            btnCancelChangesEmployee.BackColor = Color.DimGray;
```

Comment:
Please, keep colors consistent . If you want this color the cancel button in the ticket details and add/edit ticket also need to be changed.